tt_eager/tt_dnn/op_library/composite/composite_ops.cpp

```diff
@@ -831,19 +832,38 @@ Tensor _addcdiv(
     Tensor result = add(input_a, t_factor, std::nullopt, output_mem_config);
     Tensor t_inf = full_like(input_a, std::numeric_limits<float>::infinity(), output_mem_config);
     Tensor t_nan = full_like(input_a, std::nanf(""), output_mem_config);
-    return where(eqz(input_c, output_mem_config),
+
+    if(output_tensor.has_value())
+    {
+        output_tensor = where(eqz(input_c, output_mem_config),
                 ( value == 0 ) ? t_nan : where(eqz(input_b, output_mem_config),
                                             t_nan ,
                                             mul(t_inf, sign(input_b, output_mem_config), std::nullopt, output_mem_config), output_mem_config) ,
                 result, output_mem_config);
```

Comment:
Shouldn’t it be like this?

```suggestion
        output_tensor = where(eqz(input_c, output_mem_config),
                ( value == 0 ) ? t_nan : where(eqz(input_b, output_mem_config),
                                            t_nan ,
                                            mul(t_inf, sign(input_b, output_mem_config), std::nullopt, output_mem_config), output_mem_config) ,
                result, output_tensor, output_mem_config);
```